packages/css-engine/package.json

```diff
@@ -28,6 +28,7 @@
     "@types/hyphenate-style-name": "^1.0.0",
     "@types/react": "^17.0.24",
     "@types/react-dom": "^17.0.9",
+    "@webstudio-is/asset-uploader": "workspace:^",
```

Comment:
that's an additional dependency you don't want to have in css-engine

Reply:
It's dev dep for tests